Source/Android/app/src/main/res/layout/fragment_ingame_menu.xml

```diff
@@ -1,33 +1,36 @@
 <?xml version="1.0" encoding="utf-8"?>
-<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
-              xmlns:tools="http://schemas.android.com/tools"
-              android:orientation="vertical"
-              android:layout_width="match_parent"
-              android:layout_height="match_parent"
-              android:paddingBottom="16dp"
-              android:background="?attr/colorSurface"
-              tools:layout_width="250dp">
+<LinearLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:orientation="vertical"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="?attr/colorSurface"
+    tools:layout_width="250dp">
 
     <TextView
         android:id="@+id/text_game_title"
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:layout_marginBottom="16dp"
         android:layout_marginHorizontal="32dp"
-        android:layout_marginTop="32dp"
+        android:layout_marginVertical="24dp"
         android:ellipsize="end"
         android:letterSpacing="0"
         android:maxLines="@integer/game_title_lines"
         android:textSize="20sp"
         android:textColor="?attr/colorOnSurface"
         tools:text="The Legend of Zelda: The Wind Waker" />
 
+    <com.google.android.material.divider.MaterialDivider
+        android:id="@+id/divider"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content" />
+
     <ScrollView
         android:layout_width="match_parent"
         android:layout_height="0dp"
         android:layout_weight="1"
-        android:scrollbarSize="8dp"
-        android:fadeScrollbars="false">
```

Comment:
Please bring this back. While the scrollbar doesn't look great, before I added this we had lots of trouble with people not realizing that there were more options in the list.

Reply:
Ah ok